libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,58 @@
+import { AnimeTypeDto } from './amime-type.dto';
+import { AnimeStatusDto } from './anime-status.dto';
+
+/** Anime Dto. */
+export type AnimeDto = {
+
+	/** Id of anime. */
+	readonly id: number;
+
+	/** Date that anime was created. */
+	readonly created: string;
+
+	/** Last modified time. */
+	readonly modified: string;
+
+	/** Title in English. */
+	readonly title_eng: string;
+
+	/** Title in Japanese. */
+	readonly title_jpn: string;
+
+	/** Cover image of the anime. */
+	readonly image: string;
+
+	/** Broadcast time frame from start to end of a anime. */
+	readonly aired: {
+
+		/**
+		 * Start Date.
+		 * @example 2023-07-09T07:06:38.596236Z.
+		 * */
+		readonly start: string | null;
+
+		/**
+		 * End Date.
+		 * @example 2023-07-09T07:06:38.596236Z.
+		 * */
+		readonly end: string | null;
+	};
```

Comment:
Let's use ~~BroadCastedTimeDto~~ `DateRangeDto` here to reduce code duplication:
```suggestion
	readonly aired: DateRangeDto;
```